src/rust/engine/process_execution/src/lib.rs

```diff
@@ -86,6 +86,16 @@ impl TryFrom<&String> for Platform {
   }
 }
 
+impl From<Platform> for String {
```

Comment:
This looks like a PlatformConstraint maybe? Can update the naming in a followup.

Reply:
Sort of yes. The python side enum is called PlatformConstraint now, to distringuish it from the original Platform type, but on the rust side there is only this enum.